recipe/meta.yaml

```diff
@@ -41,8 +41,6 @@ requirements:
     - ipopt
 
 test:
-  requires:
-    - toolchain
```

Comment:
Probably need to add some `compiler`s in the `toolchain`s place. Do you know what compilation is happening in the test phase?

Reply:
Nothing should be getting compiled... Currently the issue seems to be popping up in the openblas requirement:
mac:
`Unsatisfiable dependencies for platform osx-64: {'openblas=0.3.3', "openblas[version='0.2.18|0.2.18.*']"}`
linux:
`Unsatisfiable dependencies for platform linux-64: {'openblas=0.3.3', "openblas[version='0.2.18|0.2.18.*']"}`